distributed/preloading.py

```diff
@@ -0,0 +1,87 @@
+import atexit
+import shutil
+import logging
+import os
+import sys
+from importlib import import_module
+
+from .compatibility import reload, invalidate_caches, cache_from_source
+
+
+logger = logging.getLogger(__name__)
+
+
+def load_file(path):
```

Comment:
A name more specific than `load_file` would be welcome IMO. `import_file` perhaps?